equinix/resource_fabric_cloud_router.go

```diff
@@ -56,6 +57,23 @@ func fabricCloudRouterProjectSch() map[string]*schema.Schema {
 	}
 }
 
+func fabricMarketplaceSubscriptionSch() map[string]*schema.Schema {
+	return map[string]*schema.Schema{
+		"type": {
+			Type:        schema.TypeString,
+			Computed:    true,
```

Comment:
Just double checking, you can send type and uuid in the request AND they will be returned in the response if not provided, correct? We need to be spot on with attribute behaviors to prepare for terraform-plugin-framework migration when we move to tf-plugin-protocol v6